src/components/App/App.jsx

```diff
@@ -67,6 +72,7 @@ class App extends React.Component {
     const github = new GitHub()
     github.getCurrentUser().then(user => {
       this.setState({ user })
+      this.onUserLoad(user.login)
     }).catch(error => {
```

Comment:
Would you instead do:

```javascript
github.getCurrentUser().then(u => this.onUserLoad(u)).`
```

and move the `this.setState({ user })` into `onUserLoad`?